.github/workflows/dev.workflow.yml

```diff
@@ -0,0 +1,57 @@
+name: Java CI
+
+on:
+  push:
+    branches:
+      - develop
+      - 'feature/**'
+      - 'bugfix/**'
+
+jobs:
+  prep:
+    runs-on: ubuntu-latest
+    outputs:
+      rev: ${{ steps.shortrev.outputs.rev }}
+      ver: ${{ steps.pversion.outputs.ver }}
+      date: ${{ steps.date.outputs.date }}
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up JDK 1.8 & run tests & generate artifacts
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Please follow the latest update here for your file java 8 instead of 1.8 and other changes
https://avi-spl.atlassian.net/wiki/spaces/SYM/pages/3012427802/GitHub+Actions+for+Adapters

Reply:
I fixed it